backend/src/resolvers/socialMedia.spec.js

```diff
@@ -30,20 +39,63 @@ describe('CreateSocialMedia', () => {
     await factory.cleanDatabase()
   })
 
+  describe('unauthenticated', () => {
+    it('throws authorization error', async () => {
+      client = new GraphQLClient(host)
+      const variables = { url: 'http://nsosp.org' }
+      await expect(
+        client.request(mutationC, variables)
+      ).rejects.toThrow('Not Authorised')
+    })
+  })
+
   describe('authenticated', () => {
     beforeEach(async () => {
       headers = await login({ email: 'test@example.org', password: '1234' })
       client = new GraphQLClient(host, { headers })
     })
 
+    it('creates social media with correct URL', async () => {
+      const variables = { url: 'http://nsosp.org' }
+      await expect(
+        client.request(mutationC, variables)
+      ).resolves.toEqual(expect.objectContaining({
+        CreateSocialMedia: {
+          id: expect.any(String),
+          url: 'http://nsosp.org'
+        }
+      }))
+    })
+
+    it('deletes social media', async () => {
+      const creationVariables = { url: 'http://nsosp.org' }
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-05-03T10:36:56Z" title="Friday, May 3rd 2019, 12:36:56 pm +02:00">May 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -30,20 +39,63 @@ describe('CreateSocialMedia', () => {
     await factory.cleanDatabase()
   })
 
+  describe('unauthenticated', () => {
+    it('throws authorization error', async () => {
+      client = new GraphQLClient(host)
+      const variables = { url: 'http://nsosp.org' }
+      await expect(
+        client.request(mutationC, variables)
+      ).rejects.toThrow('Not Authorised')
+    })
+  })
+
   describe('authenticated', () => {
     beforeEach(async () => {
       headers = await login({ email: 'test@example.org', password: '1234' })
       client = new GraphQLClient(host, { headers })
     })
 
+    it('creates social media with correct URL', async () => {
+      const variables = { url: 'http://nsosp.org' }
+      await expect(
+        client.request(mutationC, variables)
+      ).resolves.toEqual(expect.objectContaining({
+        CreateSocialMedia: {
+          id: expect.any(String),
+          url: 'http://nsosp.org'
+        }
+      }))
+    })
+
+    it('deletes social media', async () => {
+      const variablesC = { url: 'http://nsosp.org' }
```


maybe, we could find a more descriptive name? 
what about `deletionVariables`

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-05-06T13:22:31Z" title="Monday, May 6th 2019, 3:22:31 pm +02:00">May 6, 2019</time>_

---


Thank you @mattwr18 for the hint. I have done.